plugins/systemstat.koplugin/main.lua

```diff
@@ -16,8 +16,10 @@ local SystemStat = {
 }
 
 function SystemStat:init()
-    if Device:isCervantes() or Device:isKobo() or Device:isPocketBook() then
+    if Device:isCervantes() or Device:isPocketBook() then
         self.storage_filter = "mmcblk"
+    elseif Device:isKobo() then
+        self.storage_filter = "/mnt/"
```

Comment:
```self.storage_filter = = ' /mnt/onboard$' -e ' /mnt/sdcard$'```

Reply:
Assuming we only care about default storage locations, that is ;).

Reply:
Seems a bit too exclusive to me. :-P

Reply:
Kobo user data is only mounted in /mnt, so i figured to set the filter on /mnt/.
Its sufficient for a standard Kobo and filters out any other mounts (loop devices, binds)

Reply:
I'd keep the space, at least, to match on mountpoint and not blockdevice, on the off-chance the syntax gets wonky one day ;).

Reply:
Keeping the space =P